### DISCUSSION TITLE
New version release? - Solved

### DISCUSSION THREAD

**Discussion by mrblasco** [EUSurvey/EUSURVEY#482] [General]
Hi,

The latest version of EUSurvey is dated November 2020.  When do you expect to release the next version?

Thanks

**Comment by LMA-EUSurvey** [comment]
Dear  mrblasco, 

You can always download the newest version of EUSURVEY from GitHub. We have a new version from this year.
If you wish to have access to the release notes, then you can find them using the following link:  https://circabc.europa.eu/ui/group/599f39d2-e0cc-4765-bfdc-c9917c931509/library/64db67b6-748e-46ed-af55-f887e94c1dab?p=3&n=10&sort=created_ASC 

You will find the newest release e note from 25th January 2022.

Best regards
EUSurvey Support

**Reply by mrblasco** [reply]
Hi thanks for your reply. However, the Github release is not up to date (it still marks the 2020 version as the latest release) and the versions listed on the CIRCABC website do not show what changes from one version to the other -- or at least I wasn't able to find the release notes. Could you please point me to release notes with some information about the changes for the latest version?  Thanks in advance

![Github release information looks outdated](https://user-images.githubusercontent.com/4143995/161495710-50c75a03-3587-4e74-8449-5a46ea24c3a2.png)

**Comment by LMA-EUSurvey** [comment]
Dear Mrblasco,

In CIRCABC platform we keep all the release note for EUSURVEY. You can access all the latest release notes here: https://circabc.europa.eu/ui/group/599f39d2-e0cc-4765-bfdc-c9917c931509/library/64db67b6-748e-46ed-af55-f887e94c1dab?p=3&n=10&sort=created_ASC 

Let me know if this helps.
Best regards
EUSurvey Support

**Reply by mrblasco** [reply]
Thanks. Problem solved.  The notes are in PDF. I didn't notice the icon for downloading the PDF.
